js/my-github.js

```diff
@@ -1,2 +1,35 @@
 // Write code here to communicate with Github
+function getRepos(username){
+    document.getElementById('repos-list').innerHTML = '';
 
+    fetch("https://api.github.com/users/" + username + "/repos").then(
+        function(response) {
+            return response.json();
+        }
+    ).then(
+        function(repos) {
+            reposArray = [];
+            for (i = 0; i < repos.length; i++) {
+            reposArray = reposArray.concat([[repos[i]["name"], repos[i]['html_url']]]);
+            }
+            return reposArray;
+        }
+        
+    ).then(function(reposArr) {
+        for (i = 0; i < reposArr.length; i++) {
+            var liTag = document.createElement('li');
+            var aTag = document.createElement('a');
+            aTag.innerHTML = reposArr[i][0];
+            aTag.href = reposArr[i][1];
+            liTag.appendChild(aTag);
+            document.getElementById('repos-list').appendChild(liTag);
```

Comment:
You're making two passes over the array of repos - one to create an array of subarrays with only two elements, and then looping over this array to build the elements. Why not use a single pass with `forEach` and add the elments directly from the original data?